tests/kernel/nfs_stress_ng.pm

```diff
@@ -15,6 +19,33 @@ use utils;
 use registration;
 use version_utils 'is_sle';
 
+sub check_nfs_mounts {
+    my @paths = @_;
+
+    my $output = script_output("mount");
+
+    my @mounts;
+    foreach my $line (split /\n/, $output) {
+        # match only nfs mount lines
+        next unless $line =~ /\btype\s+nfs\d?\b/;
+
+        # capture the mountpoint /path from the line
+        push @mounts, $1 if $line =~ /\son\s+(\/\S+)\s+type/;
```

Comment:
Now that I think about it, this doesn't verify that a specific path is really on NFS because you may have multiple mounts on the same mountpoint. Do this instead:
```
my %mounts;

foreach my $line (split /\n/, $output) {
    $mounts{$1} = $2 if $line =~ /\son\s+(\/\S+)\s+type\s+(\S+)/;
}
```

Then check that `$mounts{$path} =~ /^nfs\d+/`.